Только в ./linux-6.5.2-custom/arch/x86/boot/compressed: drivers
Только в ./linux-6.5.2-custom/arch/x86/include: generated
Только в ./linux-6.5.2-custom/: .config
Только в ./linux-6.5.2-custom/: .config.old
Только в ./linux-6.5.2-custom/include: config
Только в ./linux-6.5.2-custom/include: generated
Только в ./linux-6.5.2-custom/: keyboardPressCounter
diff -r -u ./linux-6.5.2/Makefile ./linux-6.5.2-custom/Makefile
--- ./linux-6.5.2/Makefile	2023-09-06 23:22:26.000000000 +0300
+++ ./linux-6.5.2-custom/Makefile	2023-12-20 05:25:25.051586846 +0300
@@ -751,7 +751,7 @@
 
 ifeq ($(KBUILD_EXTMOD),)
 # Objects we will link into vmlinux / subdirs we need to visit
-core-y		:=
+core-y		:= phonebook/ keyboardPressCounter/
 drivers-y	:=
 libs-y		:= lib/
 endif # KBUILD_EXTMOD
Только в ./linux-6.5.2-custom/: Module.symvers
Только в ./linux-6.5.2-custom/: phonebook
Только в ./linux-6.5.2-custom/scripts: asn1_compiler
Только в ./linux-6.5.2-custom/scripts/basic: fixdep
Только в ./linux-6.5.2-custom/scripts: kallsyms
Только в ./linux-6.5.2-custom/scripts/kconfig: conf
Только в ./linux-6.5.2-custom/scripts/mod: devicetable-offsets.h
Только в ./linux-6.5.2-custom/scripts/mod: elfconfig.h
Только в ./linux-6.5.2-custom/scripts/mod: mk_elfconfig
Только в ./linux-6.5.2-custom/scripts/mod: modpost
Только в ./linux-6.5.2-custom/scripts: module.lds
Только в ./linux-6.5.2-custom/scripts/selinux/genheaders: genheaders
Только в ./linux-6.5.2-custom/scripts/selinux/mdp: mdp
Только в ./linux-6.5.2-custom/scripts: sorttable
Только в ./linux-6.5.2-custom/security/selinux: av_permissions.h
Только в ./linux-6.5.2-custom/tools/objtool/arch/x86: lib
Только в ./linux-6.5.2-custom/tools/objtool: fixdep
Только в ./linux-6.5.2-custom/tools/objtool: libsubcmd
Только в ./linux-6.5.2-custom/tools/objtool: objtool
Только в ./linux-6.5.2-custom/: .version
